packages/stream-model/src/model.ts

```diff
@@ -158,63 +68,146 @@ async function throwReadOnlyError(): Promise<void> {
   )
 }
 
+/**
+ * Represents the relationship between an instance of this model and the controller account.
+ * 'list' means there can be many instances of this model for a single account. 'link' means
+ * there can be only one instance of this model per account (if a new instance is created it
+ * overrides the old one).
+ */
+export enum ModelAccountRelation {
+  LIST = 'list',
+  LINK = 'link',
+}
+
+/**
+ * Contents of a Model Stream.
+ */
+export interface ModelDefinition {
```

Comment:
In `@glazed/types` we have:
```
export declare type ModelDefinition = {
    name: string;
    description?: string;
    schema: JSONSchema.Object;
    accountRelation: ModelAccountRelation;
    relations?: ModelRelationsDefinition;
    views?: ModelViewsDefinition;
};
```

I think `relations` and `views` will not be implemented for the MVP, but just highlighting in case we want to use the same types from `commons` in `glazed` and `ceramic`.

Reply:
Good callout. Would be great if we can import types from Ceramic in glaze.

Reply:
yeah I think once this PR gets merged it would be great if glaze switched to importing and using these types.  I don't think anything needs to change with this PR for that to be possible

Reply:
I'd rather leave them out for now so it's clear to users that those fields can't be set yet.  I don't want people trying to make models with those fields now and have them ignored or be invalid when we eventually add those fields for real

Reply:
Sounds good, thanks :) !